client/test/multiledger.go

```diff
@@ -35,15 +35,16 @@ import (
 	"polycry.pt/poly-go/test"
 )
 
-// MultiLedgerTest is the setup of a multi-ledger test.
-type MultiLedgerTest struct {
-	C1, C2 testClient
-	L1, L2 *MockBackend
-	A1, A2 multi.Asset
+// MultiLedgerSetup is the setup of a multi-ledger test.
+type MultiLedgerSetup struct {
+	Client1, Client2               Client
```

Comment:
I think it would be better if the Client struct includes Adjudicator1 and Adjudicator2 as the initialization of the Adjudicator in the Ethereum backend is specific for the client (receiver address and txSender).